src/components/Details/Details.jsx

```diff
@@ -0,0 +1,61 @@
+import React, {useState, useEffect} from 'react';
+import {useParams} from 'react-router-dom';
+
+import Spinner from '../Spinner';
+import Detail from '../Detail';
+
+import './details.scss';
+
+function Details({ match }) {
+  const [details, setDetails] = useState([]);
+
+  const [isLoading, setIsLoading] = useState(false);
+
+  const [isError, setIsError] = useState(true);
+
+  const {eventId} = useParams();
+
+  useEffect(() => {
+    fetchItem();
+  }, []);
+
+  const fetchItem = async () => {
+    try {
+      setIsLoading(true);
+
+      const data = await fetch(
+        `https://soft.silverscreen.by:8443/wssite/webapi/event/data?filter=%7B%22event%22:%22${eventId}%22,%22city%22:%221%22%7D&extended=true`
+      );
+
+      const resultList = await data.json();
+
+      setDetails(resultList);
```

Comment:
здесь не массив, а обьект, начальное значение тоже {}
```suggestion
      setDetails(movieDetails[0]);
```

Reply:
fixed